source/funkin/ui/title/AttractState.hx

```diff
@@ -73,9 +73,15 @@ class AttractState extends MusicBeatState
     {
       vid.zIndex = 0;
       vid.bitmap.onEndReached.add(onAttractEnd);
-
       add(vid);
-      vid.play(filePath, false);
+
+      openfl.Assets.loadBytes(filePath).onComplete(function(bytes:openfl.utils.ByteArray):Void
+      {
+        if (vid.load(bytes))
+          vid.play();
+
+        onVideoStarted.dispatch();
```

Comment:
Looks good, you just forgot to remove this line you copied from VideoCutscene

Reply:
> Looks good, you just forgot to remove this line you copied from VideoCutscene

My bad